packages/api/src/routers/tx/getCount.ts

```diff
@@ -16,19 +16,6 @@ const outputSchema = z.object({
 });
 
 export async function countTxs(prisma: BlobscanPrismaClient, filters: Filters) {
-  if (!hasCustomFilters(filters)) {
```

Comment:
Same as with countBlobs:

- add comment explaining the logic we use now
- add test cases